test/controllers/static_pages_controller_test.rb

```diff
@@ -0,0 +1,23 @@
+require "test_helper"
```

Comment:
xóa hết toàn bộ file trong thư mục test nhé, các chú không phải làm phần `test`, sau sẽ học rspec thay cho test